.ci/install_clearcontainers_kernel.sh

```diff
@@ -0,0 +1,63 @@
+#!/bin/bash
+#
+# Copyright (c) 2017 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+repo_owner="clearcontainers"
+repo_name="linux"
+cc_linux_releases_url="https://github.com/${repo_owner}/${repo_name}/releases"
+
+set -e
```

Comment:
Nit: this should probably come immediately after the header comment.

Reply:
Agree, updated.